lib/cassava/client.rb

```diff
@@ -0,0 +1,166 @@
+module Cassava
+  class Client
+    attr_reader :session
+
+    # @param [Cassandra::Session] The session object
+    def initialize(session)
+      @session = session
+    end
+
+    def insert_async(table, data)
+      session.execute_async(insert_statement(table, data), :arguments => data.values)
+    end
+
+    def insert(table, data)
```

Comment:
Do add a comment it might not be a bad idea to have insert be a wrapper for multi_insert to allow batching in the future. Cassandra-rb did a decent job with this: https://github.com/cassandra-rb/cassandra/blob/master/lib/cassandra/cassandra.rb#L553